controllers/providerconfigmap/provider_configmap_controller.go

```diff
@@ -287,6 +288,21 @@ func (r *ConfigMapReconciler) Reconcile(ctx goctx.Context, req ctrl.Request) (ct
 func (r *ConfigMapReconciler) ReconcileNormal(ctx goctx.Context, cm *corev1.ConfigMap) error {
 	r.Logger.Info("Reconciling VM provider ConfigMap", "name", cm.Name, "namespace", cm.Namespace)
 
+	// We only update TKG related ContentSource/ContentLibraryProvider/ContentSourceBinding resources
+	// in provider configmap reconcile loop for now.
+	if lib.IsWCPVMImageRegistryEnabled() {
```

Comment:
I'm confused: shouldn't this be `!lib.IsWCPVMImageRegistryEnabled()` to _stop_ creating these when the FFS is enabled? My initial thought was that this whole controller isn't needed anymore when the FFS is enabled. What still needs the source or binding now because in the image lookup path AFAICT we don't depend on those CRs anymore when the FFS is enabled.

Reply:
Oh yes!!! I forgot the `!`

Reply:
I was thinking about adding the entire controller to the manager only if the FSS is disabled, but ended up keeping this controller, so that if we don't have to add it back if we want to watch the provider configMap for other resources in the future? But yah, it's not needed when the FSS is enabled. Will update this.

Reply:
Yes let's disable this whole controller if it isn't needed.